temp/prometheus-builder/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+PR_NUMBER=$1
+
+if [ -z "$PR_NUMBER" ]
+then
+	echo "ERROR:: PR NUMBER is missing in argument"
+	exit 1
+fi
+
+DIR="/go/src/github.com/prometheus/prometheus"
+
+printf "\n\n>> Fetching Pull Request\n\n"
+git clone https://github.com/prometheus/prometheus.git $DIR
+
+cd $DIR
+git fetch origin pull/$PR_NUMBER/head:pr-branch
+git checkout pr-branch
+printf "\n\n>> Creating prometheus binaries\n\n"
+make build
```

Comment:
Where is the makefile for this step?

Reply:
This uses the cloned Prometheus [Makefile](https://github.com/prometheus/prometheus/blob/master/Makefile.common#L96)

Reply:
put a copy inside the same folder and later will decide what to do with this docker image.